Iks315 notify on all status changes (#1733)
ICTU/quality-time: docs/CHANGELOG.md

- Accepting the technical debt of a metric with one or more unreachable sources would set the metric status to 'technical debt target met' instead of 'unknown'. Fixes [#1636](https://github.com/ICTU/quality-time/issues/1636).
- Collapsing an expanded metric would sometimes result in a crash of the frontend. Fixes [#1717](https://github.com/ICTU/quality-time/issues/1717).

> - Notifications can now be send for all status changes. Previously notifications where only sent out when a metric either turned red or white.

Reviewer: Status changes aren't configurable, right? So shouldn't this read "will be sent" instead of "can be sent"?

```suggestion
- Notifications will now be sent for all status changes. Previously, notifications were only sent when a metric either turned red or white.
```
